client/src/components/new-request/grey-box.vue

```diff
@@ -1,130 +1,124 @@
+          <ReserveSubmit setup="examine"
+                         id="reserve-submit-button"
+                         style="display: inline" />
+        </v-col>
+        <v-col cols="12" class="text-center" v-else>
+          <v-btn @click="cancelAnalyzeName('Tabs')">Start Over</v-btn>
+        </v-col>
+      </template>
+      <template v-else>
+        <transition :name="i === 0 ? 'fade' : '' " mode="out-in">
+          <v-row :key="changesInBaseName+designationIsFixed+'key'+i"
+                 align-content="start"
+                 class="copy-small colour-p-blue-text">
+            <!-- Header, Line 1 and Line 2-->
+            <v-col class="copy-small colour-p-blue-text"
+                   v-if="changesInBaseName && isDesignationIssueType && i === 0"
+                   cols="12">
+              You have altered the base text of your name.  You must
+              either change it back or click the magnifying glass to run a new search for your edited name.
+            </v-col>
+            <v-col class="copy-small colour-p-blue-text"
+                   v-else
+                   cols="12">
+              <p v-if="option.line1" class="ma-0 pa-0" v-html="option.line1" />
+              <p v-if="option.line2" class="ma-0 pa-0 pt-2 mb-n1" v-html="option.line2" />
+            </v-col>
 
-          <v-col class="copy-small colour-p-blue-text pt-0"
-                 v-if="designationIsFixed && isDesignationIssueType && i === 0"
-                 cols="12">
-            {{ isLastIndex ? 'No additional issues were identified with your name.  You may proceed.' :
-            'Please click the "Next Issue" button to continue' }}
-          </v-col>
-          <v-col class="copy-small colour-p-blue-text pt-0"
-                 v-else-if="changesInBaseName && isDesignationIssueType && i === 0"
-                 cols="12">
-            You have altered the base text of your name.  You must
-            either change it back or click the magnifying glass to run a new search for your edited name.
-          </v-col>
-          <v-col class="copy-small colour-p-blue-text pt-0"
-                 v-else
-                 cols="12">
-            <p v-if="option.line1" class="ma-0 pa-0" v-html="option.line1" />
-            <p v-if="option.line2" class="ma-0 pa-0 pt-2" v-html="option.line2" />
-          </v-col>
+            <!-- NAME/DESIGNATION-MANIPULATING OPTION BOXES -->
+            <template v-if="isDesignationIssueType">
+              <transition name="fade"
+                          mode="out-in">
+                <v-col v-if="!designationIsFixed && !changesInBaseName && i === 0"
+                       class="text-center designation-error-col"
+                       key="designation-error-col">
+                  <template v-if="Array.isArray(designations) && designations.length > 0">
+                    <div class="designation-buttons pa-0">
+                      <button :id="'designation-'+d"
```

Comment:
It would be nice to have "button" or "btn" as part of the id.

Reply:
is this something we would actually like to change? Because the vuetify tag was called 'v-btn' i'm pretty sure anywhere/everywhere in the app where I had to name a class, or ID, or make a method or getter refer to a button, I wrote it "btn" unfortunately.

Reply:
Whether it's a Vuetify button or a plain HTML button, it's still ultimately rendered as a button element.

I'm thinking it would make it easier, in a SASS file or a test file, to know that this element is a button, if the name contained "btn", eg. designation-foo-btn. And it would be consistent with other button IDs.

Reply:
sure